src/MassTransit.QuartzIntegration.Tests/Recurring_Specs.cs

```diff
@@ -104,6 +104,8 @@ public MySchedule()
 
                 StartTime = DateTime.Now + TimeSpan.FromSeconds(3);
                 EndTime = StartTime + TimeSpan.FromSeconds(7);
+
+                Description = "my description";

```

Comment:
I'm not sure it's worth doing that effort -- if you've tested this and you're comfortable with it, the impact is small enough I think we could just go with it. There were a couple of formatting fixes to do then I think we could give this a go.